pkg/executables/cmk_test.go

```diff
@@ -19,23 +19,46 @@ import (
 )
 
 const (
-	cmkConfigFileName = "cmk_tmp.ini"
-	accountName       = "account1"
-	rootDomain        = "ROOT"
-	rootDomainId      = "5300cdac-74d5-11ec-8696-c81f66d3e965"
-	domain            = "foo/domain1"
-	domainName        = "domain1"
-	domainId          = "7700cdac-74d5-11ec-8696-c81f66d3e965"
-	domain2           = "foo/bar/domain1"
-	domain2Name       = "domain1"
-	domain2Id         = "8800cdac-74d5-11ec-8696-c81f66d3e965"
-	zoneId            = "4e3b338d-87a6-4189-b931-a1747edeea8f"
+	cmkConfigFileName  = "cmk_test_name.ini"
+	cmkConfigFileName2 = "cmk_test_name_2.ini"
+	accountName        = "account1"
+	rootDomain         = "ROOT"
+	rootDomainId       = "5300cdac-74d5-11ec-8696-c81f66d3e965"
+	domain             = "foo/domain1"
+	domainName         = "domain1"
+	domainId           = "7700cdac-74d5-11ec-8696-c81f66d3e965"
+	domain2            = "foo/bar/domain1"
+	domain2Name        = "domain1"
+	domain2Id          = "8800cdac-74d5-11ec-8696-c81f66d3e965"
+	zoneId             = "4e3b338d-87a6-4189-b931-a1747edeea8f"
 )
 
 var execConfig = decoder.CloudStackExecConfig{
-	ApiKey:        "test",
-	SecretKey:     "test",
-	ManagementUrl: "http://1.1.1.1:8080/client/api",
+	Profiles: []decoder.CloudStackProfileConfig{
+		{
+			Name:          "test_name",
+			ApiKey:        "test",
+			SecretKey:     "test",
+			ManagementUrl: "http://1.1.1.1:8080/client/api",
+		},
+	},
+}
+
+var execConfigWithMultipleProfiles = decoder.CloudStackExecConfig{
+	Profiles: []decoder.CloudStackProfileConfig{
+		{
+			Name:          "test_name",
+			ApiKey:        "test",
+			SecretKey:     "test",
+			ManagementUrl: "http://1.1.1.1:8080/client/api",
+		},
```

Comment:
```suggestion
		...execConfig.Profiles[0],
```

Would this be easier? Doesn't matter much

Reply:
Good idea!